### PR TITLE
Wrong GPO

### PR DESCRIPTION
**Pull request by super0xbad1dea**
LME-Sysmon-Task is not part of LME-WCE-*. Please correct the link to LME-Sysmon-Task (https://github.com/ukncsc/lme/tree/master/Chapter%202%20Files/GPO%20Deployment)

### CODE REVIEW

**Comment by duncan-ncc** [comment]
Thanks for your time, This is obviously is an issue. I would propose that the link was relative rather than fully qualified to ensure compatibility with forks and branches.

**Review by duncan-ncc — CHANGES_REQUESTED** [review]
As discussed in the comments, I propose you changing the URL to relative.

**Comment by super0xbad1dea** [comment]
I propose that, instead of URL you zip the GPO folder like you did with the GPO-files for the logging configuration on client and server. Makes it more comfortable to download a zipped file.

File: docs/chapter2.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Figure 5: Sysvol File Layout
 ### 2.2.2 - Scheduled task GPO Policy
 This section sets up a scheduled task to run update.bat (stored on a network folder), distributed through Group Policy.
 
-Import the [LME-Sysmon-Task](/Chapter%201%20Files/lme_gpo_for_windows.zip) GPO into group policy management and link the object to a test Organisational Unit (OU). Once the GPO is confirmed as working in your environment then you can link the GPO to a larger OU to deploy LME further.
+Import the [LME-Sysmon-Task](https://github.com/ukncsc/lme/tree/master/Chapter%202%20Files/GPO%20Deployment) GPO into group policy management and link the object to a test Organisational Unit (OU). Once the GPO is confirmed as working in your environment then you can link the GPO to a larger OU to deploy LME further.
```
Comment (super0xbad1dea) [inline docs/chapter2.md:51]:
```suggestion
Import the [LME-Sysmon-Task](/Chapter%202%20Files/GPO%20Deployment) GPO into group policy management and link the object to a test Organisational Unit (OU). Once the GPO is confirmed as working in your environment then you can link the GPO to a larger OU to deploy LME further.
```

**Review by super0xbad1dea — COMMENTED** [review]
relative path

File: docs/chapter2.md (hunk 1 above)
Comment (duncan-ncc) [reply docs/chapter2.md:51]: Looks good. I am unable to commit the suggestion automatically on GitHub though.

**Review by duncan-ncc — APPROVED** [review]

**Comment by duncan-ncc** [comment]
I am going to close this PR as the URL in question has been changed as part of https://github.com/ukncsc/lme/commit/7168abd380e92d4c557425f15a4c021b56556f14.
Thanks for your contribution.

**Pull request closed without merging**